### ISSUE TITLE
Hot keys for time navigation

### ISSUE THREAD

**Issue by artenax** [DeaDBeeF-Player/deadbeef#2920]
Hello.
I was surprised to find that deadbeef has no hotkeys for track navigation. Implement this, please. For any reasonable time.
Audacious has one, but it requires customization.

What is it for?
Imagine, for example, a blind or visually impaired person listening to audiobooks (several hours long) and wants to skip already listened before. Or just a person lying on a sofa with headphones, keyboard next to him and a long way to reach the mouse. There are hot keys to switch to the next/previous track, but for navigation within the track there is none. But the track can be large, in the case of audiobooks.

**Comment by Oleksiy-Yakovenko** [comment]
There are some configurable actions in the preferences, which can be assigned to hotkeys:
They work at granularity of 1sec, 5sec, 1% and 5%, forward or backward. I'm not sure whether this is what you are looking for though.
A little hard to find, but they are under Main/Playback category

<img width="584" alt="image" src="https://user-images.githubusercontent.com/1895698/219900189-c9272ba3-e364-4560-9ec9-9658185859e8.png">

**Comment by artenax** [comment]
Yes, that's it. Thank you. I hadn't seen these settings before. 1% and 5% are good values.

By the way, in Audacious only the back/forward keys work, which rewind the files (lasting several hours) rather slowly. foobar lets set an arbitrary time (or there are some steps, I can't remember, but it requires customization). And video players like VLC.

**Comment by kuba160** [comment]
There is a plugin that adds 2 extra seek hotkeys with configurable second/percentage value: https://github.com/kuba160/ddb_cust_seek

**Comment by artenax** [comment]
kuba160
Thanks. I compiled the plugin, but how do I switch to time mode only? Or only custom percentages are supported?

**Comment by kuba160** [comment]
Switching the checkbox off in plugin settings above value configuration should seek given amount of seconds, does it not work as intended?

**Comment by artenax** [comment]
I have these settings:

![01](https://user-images.githubusercontent.com/107228652/219905310-3bda9901-3e0c-41e1-aeda-b8adfc7ff777.png)

![02](https://user-images.githubusercontent.com/107228652/219905315-a61f24c9-1e15-4921-8cb8-0cf2b940a41a.png)

**Comment by kuba160** [comment]
Checkbox "Preset 1: Seconds/Percentage" switches between time mode (unchecked) and percentage mode (checked). You have 2 independent presets that can be configured separately. I updated the plugin to use the drop-down menu instead, I don't know why I didn't do that in first place. Hopefully this clears the confusion.

**Comment by artenax** [comment]
kuba160
Yes, I get it now, it works. Thanks.

![03](https://user-images.githubusercontent.com/107228652/219953453-53c7cf0e-8085-463a-92f6-9f57124261c2.png)
